git-link.el

```diff
@@ -187,8 +187,8 @@
   :group 'git-link)
 
 (defcustom git-link-open-in-browser nil
-  "If non-nil also open link in browser via `browse-url'."
-  :type 'boolean
+  "If non-nil also open link with this `browse-url-*' function."
+  :type 'function
```

Comment:
Possible to be `'function` and `'boolean`? Allowing `t` is nice for those that don't care.

Reply:
> Allowing t is nice for those that don't care.

How the URL is opened.

Reply:
I presume `t` is still allowed. Because the value cell will be `t` and the function cell will be void. Don't quote me on that :smiley:

Reply:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Composite-Types.html
There is a choice type mentioned in the link. I don't think anyone has ever used it though :blush: 
`:type '(choice boolean function)`

Reply:
Okay, according to [the docs](https://www.gnu.org/software/emacs/manual/html_node/elisp/Composite-Types.html) we can do this: 

```el
:type '(list function boolean)
```

Reply:
`list` means function and boolean are together in a list.

Reply:
Ah yes I see. `choice` sounds good.